engine/table/src/main/java/io/deephaven/engine/table/impl/SliceLikeOperation.java

```diff
@@ -36,6 +36,7 @@ public static SliceLikeOperation headPct(final QueryTable parent, final double p
                 0, 0, true) {
             @Override
             protected long getLastPositionExclusive() {
+                // Assuming percent is not negative here
```

Comment:
We aren't assuming, we know.  Suggest:
`// Already verified that percent is not negative`

Reply:
Done